.clang-format

```diff
@@ -0,0 +1,7 @@
+BasedOnStyle: Google
+ColumnLimit: 130
```

Comment:
@w-bruns I chose this number semi-randomly: the default is 80, I think but my feeling is that this is a bit low on modern systems, where one has much bigger screens. On the other hand, I feel that some limit is needed, else some stuff gets hard to read. But I am happy to change this something else, if you want it

Reply:
I think 130 is o.k.